rotkehlchen/db/schema.py

```diff
@@ -128,6 +128,8 @@
 INSERT OR IGNORE INTO location(location, seq) VALUES ('u', 53);
 /* Binance Smart Chain */
 INSERT OR IGNORE INTO location(location, seq) VALUES ('v', 54);
+/* Hyperliquid */
+INSERT OR IGNORE INTO location(location, seq) VALUES ('w', 55);
```

Comment:
hmmmm No I don't think this is a good idea. We do not use protocols as locations.